api/migrations/versions/742a2a556eb9_.py

```diff
@@ -0,0 +1,28 @@
+"""Add languages column to organisation
+
+Revision ID: 742a2a556eb9
+Revises: de12a6f382c1
+Create Date: 2020-07-23 19:25:52.384921
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '742a2a556eb9'
+down_revision = 'de12a6f382c1'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('organisation', sa.Column('languages', sa.JSON(), nullable=True))
+    op.execute("""UPDATE organisation SET languages = '[{"code": "en", "description": "English"}]'""")
```

Comment:
@avishkar58 Is AI4D going to be a separate organisation? If I read the previous migration scripts correctly, it looks like there is an AI4D Event under the Deep Learning Indaba? Just wondering in case we need to set it to English and French.

Reply:
Yup AI4D will be a separate organisation, the AI4D event under the Indaba will remain English only :)